### PR TITLE
Add patchless profile

### PR DESCRIPTION
**Pull request by utopiabound**
Requires whamcloud/lustre-ldiskfs-zfs#15

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

**Comment by jgrund** [comment]
Can you update the SSI testing to use this profile?

**Comment by utopiabound** [comment]
> Can you update the SSI testing to use this profile?

This patch needs whamcloud/lustre-ldiskfs-zfs#15 to land and be built for copr devel.

**Comment by jgrund** [comment]
> This patch needs whamcloud/lustre-ldiskfs-zfs#15 to land and be built for copr devel.

This has landed/\.

**Comment by jgrund** [comment]
On SSI this is returning:

```
remote_command[vm5]: 'i=0; while ! zpool create zfs_pool_scsi0QEMU_QEMU_HARDDISK_target1 -o cachefile=none -o multihost=on /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_target1 && [ $i -lt 10 ]; do sleep 1; let i+=1; done; exit ${PIPESTATUS[0]}' failed:'' stderr:'bash: zpool: command not found
```

File: lustre-server-patchless.repo [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[lustre]
+name=Lustre Server
+baseurl=https://build.whamcloud.com/jobs-pub/lustre-b2_12/configurations/axis-arch/x86_64/axis-build_type/server/axis-distro/el7.6/axis-ib_stack/inkernel/lastStable/archive/artifacts/
```
Comment (jgrund) [inline lustre-server-patchless.repo]:
```suggestion
baseurl=https://build.whamcloud.com/jobs-pub/lustre-b2_12-patchless/configurations/axis-arch/x86_64/axis-build_type/server/axis-distro/el7.6/axis-ib_stack/inkernel/lastStable/archive/artifacts/
```

**Pull request merged**